src/main/java/Marites.java

```diff
@@ -0,0 +1,198 @@
+    private static final String COMMAND_ADD_DEADLINE_TAG = "/by";
+    private static final String COMMAND_ADD_EVENT_TAG = "/at";
+
+    private static final String LIST_TASK_ITEM_FORMAT_STRING = "%d. %s%n";
+    private static final String ADD_TASK_FORMAT_STRING =
+        "Alright, task added:%n  %s%nYour list currently has %d tasks.%n";
+    private static final String MARK_DONE_MESSAGE = "Good job on getting this done!";
+    private static final String MARK_UNDONE_MESSAGE = "Okay, I've marked this as not yet done:";
+    private static final String SET_TASK_STATUS_FORMAT_STRING = "%s%n    %s%n";
+
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+    /** The task list being tracked by Marites. */
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+
+
+
+    public static void main(String[] args) {
+        printIntroduction();
+        while (true) {
+            String userInput = SCANNER.nextLine();
+            String commandFeedback = processUserCommand(userInput);
+            showFeedback(commandFeedback);
+        }
+    }
+
+    /* OUTPUT */
+
+    /**
+     * Prints a separator string.
+     */
+    private static void printSeparator() {
+        System.out.println("========================================");
+    }
+
+    /**
+     * Prints an introductory message.
+     */
+    private static void printIntroduction() {
+        System.out.println(INTRO_MESSAGE);
+        printSeparator();
+    }
+    /**
+     * Prints an exit message.
+     */
+    private static void printExitMessage() {
+        System.out.println(EXIT_MESSAGE);
+        printSeparator();
+    }
+
+    /**
+     * Prints a feedback string from an executed command.
+     * @param feedback The feedback string to execute.
+     */
+    private static void showFeedback(String feedback) {
+        printSeparator();
+        System.out.print(feedback);
+        printSeparator();
+    }
+
+    /**
+     * Get the type of command issued.
+     * @param userInput The user's input
+     * @return The command type.
+     */
+    private static String getCommandType(String userInput) {
+        String[] tokens = userInput.split("\\s");
+        return tokens[0];
+    }
+
+    /**
+     * Parses a user's command, and executes it.
+     * @param userInput A line of the user's input.
+     * @return Output that must be passed back to the user.
+     */
+    private static String processUserCommand(String userInput) {
+        switch (getCommandType(userInput)) {
+        case COMMAND_EXIT:
+            return executeExit();
+        case COMMAND_LIST:
+            return executeListTasks();
+        case COMMAND_MARK:
+            return executeSetTaskStatus(userInput, true);
+        case COMMAND_UNMARK:
+            return executeSetTaskStatus(userInput, false);
+        case COMMAND_ADD_TODO:
+        case COMMAND_ADD_DEADLINE:
+        case COMMAND_ADD_EVENT:
+            return executeAddTask(userInput);
```

Comment:
I suggest adding a "// Fallthrough" comment for the cases that falls through to align with given coding standards

Reply:
Thanks for catching that. I've added such a comment.